src/scripts/ci/pylint_plugins/gh_actions_reporter/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""
+Pylint reporter for GitHub Actions
+"""
+from __future__ import annotations
```

Comment:
What this is doing seems quite simple so we should just do it ourselves. And especially NAK on anything that is not either strictly BSD-2 or public domain.

I know it seems like MIT vs BSD-2 is basically the same, but there are many organizations where us including a second LICENSE file, even for something not used in the final binary, will require an expensive review process. Seems silly I know! But this is a thing. If this seems incredible, be grateful.

If you want I'll do this part. I have barely looked at this code but given the knowledge that pylint has an API is sufficient.